uint64 time
uint32 steps
string team
uint32 vision_range
